src/main/backend/database/migrations/20171006232944_v2.js

```diff
@@ -1,14 +1,22 @@
 exports.up = (knex, Promise) => {
   return Promise.all([
-    knex.schema.alterTable('categories', (table) => {
-      table.timestamp('created_at').notNullable().defaultTo('2013-12-31 12:17:10.0');
-      table.timestamp('updated_at').notNullable().defaultTo('2013-12-31 12:17:10.0')
+    knex.schema.alterTable('categories', table => {
```

Comment:
I would like to keep `(table)`, because this:

```js
(table) => {}
(table, column) => { }
```

Is more consistent than:

```js
table => {}
(table, column) => { }
```

Reply:
I'm afraid this is not possible dear Panda 🐼